src/libraries/pkg/test/testPackages.proj

```diff
@@ -3,19 +3,26 @@
 
   <ItemGroup>
     <PackagesToTest Condition="'$(PackagesToTest)' != ''" Include="$(PackagesToTest)" />
-    <!-- transport package not expected to be closure complete  -->
-    <ExcludePackages Include="Microsoft.AspNetCore.Internal.Transport" />
-    <ExcludePackages Include="$(ExcludePackages)" />
```

Comment:
>ExcludePackages [](start = 5, length = 15)

Sorry for joining the party late :smile: Should we keep this line in case folks want to pass in extra exclude through command line like:`/p:ExcludePackages=...`. Not sure how used that was but it was there so I assume somebody was using it.

Reply:
I don't think anyone was using it. I myself didn't know of this switch until very recently.

Reply:
i always end up using ```packagesToTest``` in order to narrow down the noise from other projects and debug the failing package.